tox.ini

```diff
@@ -1,24 +1,21 @@
 [tox]
-passenv = 3.6.0
```

Comment:
That is plain wrong... `passenv` is used to pass environment variables into the tox environment, not to set any version number.